frontend/views/containers/notifications/NotificationCard.vue

```diff
@@ -56,7 +56,7 @@ export default {
       this.toggleTooltip()
     },
     markAllNotificationsAsRead () {
-      sbp('gi.notifications/markAllAsRead')
+      sbp('gi.notifications/markAllAsRead', this.currentGroupId)
```

Comment:
This is needed because the SW doesn't know the current group ID.

Reply:
See this addition: `sbp('okTurtles.events/emit', NOTIFICATION_STATUS_LOADED, status)`